### ISSUE TITLE
Add new card to top or bottom of a list

### ISSUE THREAD

**Issue by Latechneuse** [stefan-niedermann/nextcloud-deck#1597]
### Is your feature request related to a problem? Please describe.

When managing long lists, it could be useful to add a new card to the top of a list. Currently, new cards will be appended to the bottom of the list. Dragging the card to the top can be quite cumbersome if you have a long list. 
I usually order cards by priority within each list. This is why it often happens that a card needs to be added to the top. 

### Describe the solution you'd like

In the new card dialog, two separate buttons could be offered: "add to the top" "dd to the bottom".

**Comment by stefan-niedermann** [comment]
While I usually refer to the (not by us developed) Deck server app, which we aim to align and being consistent with (see https://github.com/nextcloud/deck/issues/1961), I'll keep this open since we already have a pre-create dialog with the title where we easily could add a switch `Prepend at top` or something similar. Keeping open therefore.

**Comment by Latechneuse** [comment]
Yes, I was thinking of the pre-create dialog as the best place for such a button. 
Thank you for considering this improvement. (And for your amazing work.)
